File: Rule.py
num = int(input("Enter a number: "))

last_digit = num % 10
last_two = num % 100

if last_digit == 0 and last_two % 4 == 0:
    print(num, "is divisible by 20")
else:
    print(num, "is not divisible by 20")

File: Rule.cpp
#include <iostream>
using namespace std;

int main() {

    int num;

    cout<<"Enter a number: ";
    cin>>num;

    int lastDigit = num % 10;
    int lastTwo = num % 100;

    if(lastDigit == 0 && lastTwo % 4 == 0)
        cout<<num<<" is divisible by 20";
    else
        cout<<num<<" is not divisible by 20";

    return 0;
}
